pkg/render/logstorage.go

```diff
@@ -323,6 +331,21 @@ func (es *elasticsearchComponent) Objects() ([]runtime.Object, []runtime.Object)
 			toCreate = append(toCreate, es.elasticEnterpriseTrial())
 		}
 
+		// OIDCUsersConfigMapName and OIDCUsersSecreteName should be created only once
```

Comment:
I added comments in the controller about this, we should be passing the config map and secrets to render to be re rendered so that in the future if there's changes we need to make to any meta data it will be easily applied, and there's no worry about overriding the config map data.